open-source-license-acknowledgements-and-third-party-copyrights.md

```diff
@@ -100,7 +100,7 @@ https://github.com/vrana/adminer/blob/master/readme.txt
 ### Elasticsearch
 Image: `Docker.elastic.co/elasticsearch/elasticsearch-oss`  
 License: Apache License 2.0  
-https://github.com/elastic/elasticsearch/blob/66b5ed86f7adede8102cd4d979b9f4924e5bd837/LICENSE.txt  
+https://github.com/elastic/elasticsearch/blob/master/LICENSE.txt
```

Comment:
Linking a file in the `master` branch is not preferred, as it may get renamed in the future. We had this exact problem in #1077.

For the Docker image, we use the version `6.3.2`, so I'd link https://github.com/elastic/elasticsearch/blob/v6.3.2/LICENSE.txt

(Not get the Docker image mixed up with the PHP client library https://github.com/elastic/elasticsearch-php which we use in version `^6.0.1`)

---

Also, please don't remove the two trailing spaces as they add a newline in Markdown. The removal has broken the formatting:

![image](https://user-images.githubusercontent.com/10008612/58429665-e962bf00-80a6-11e9-9d82-a2c3525ae4b5.png)

Reply:
Hello @LukasHeinz is this still valid and will you update this PR or can we close it?